packages/babel-traverse/src/scope/index.js

```diff
@@ -864,6 +868,14 @@ export default class Scope {
   }) {
     let path = this.path;
 
+    if (path.isPattern()) {
+      // We are inside some function parameters
```

Comment:
`path.isPattern` does not implies that we are inside function parameters. It can also be an object pattern in a block statement, etc.

Reply:
In that case, the `path` would be the block statement (here `path` is `scope.path`).